pygdf/dataframe.py

```diff
@@ -620,15 +653,39 @@ def sort_index(self, ascending=True):
 
     def sort_values(self, by, ascending=True):
         """
-        Sort by values.
+        Uses parallel radixsort, which is a stable sort.
 
-        Difference from pandas:
-        * *by* must be the name of a single column.
-        * Support axis='index' only.
-        * Not supporting: inplace, kind, na_position
```

Comment:
Can we keep the difference from pandas in a `Notes` section of the docstring, similar to `join` method (line 835 in this file)?

Reply:
We have differences from Pandas in many functions, but 'sort_values' in DataFrame and Series are the only places we call it out like above

IMO we should doc the supported parameters, unless the behavior of the function is significantly different (Series naming for groupby objects, for instance).

Otherwise, for consistency, we'd need to go back and add "Difference from pandas:" blurbs for every function where we don't have some param that the Pandas equivalent function has. Personally, that would feel cluttered, and doesn't add information not already contained in the "Parameters" section.

Thoughts?

Reply:
I totally agree that not all of the differences from pandas need to be in the docstrings, I think this might be the case since sort is such a frequently used function and the `by` and `axis` differences are significant. Ao I would support keeping (or adding later) the blurb in `join/merge`, 'sort_values` and `groupby`. But I'm also fine if we end up just putting all of that in a single doc about "faq about differences from pandas". So I'll close the comment and leave it to your judgement :).